monitoring/rid_qualifier/run_locally.sh

```diff
@@ -44,21 +44,22 @@ CONFIG='--config config.json'
 RID_QUALIFIER_OPTIONS="$AUTH $CONFIG"
 
 # report.json must already exist to share correctly with the Docker container
-touch $(pwd)/monitoring/rid_qualifier/report.json
+touch "$(pwd)/monitoring/rid_qualifier/report.json"
 
 docker build \
     -f monitoring/rid_qualifier/Dockerfile \
     -t interuss/dss/rid_qualifier \
-    --build-arg version=`scripts/git/commit.sh` \
+    --build-arg version="$(scripts/git/commit.sh)" \
     monitoring
 
+# shellcheck disable=SC2086
```

Comment:
Why do we need to disable here?  Don't the quotes below fix the issue?

Reply:
If we quote the variable, the argument is passed as a single argument (escaping word splitting) and the python script fails to properly parse arguments.